server/tests/backends/fixtures/formula/special_brackets_pypika.json

```diff
@@ -0,0 +1,94 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "bigquery_pypika"
```

Comment:
Why is bigquery excluded here ? Which column has a space in its name ?

Reply:
No columns has spaces for the dataset used to test this fixture !
I assumed the user trying to encapsulate it with brackets and/or something else, and genuinely, we should never have a dataset with spaces inside columns (https://stackoverflow.com/questions/28926175/can-you-use-field-alias-with-space-in-google-big-query) that's why i added this exclusion.

Reply:
Users might still want to surround quotes in brackets (maybe because they copy and adapt a query from somewhere), even if spaces are not supported. I don't feel comfortable with having an inconsistent API, supporting brackets in some places but not everywhere